hbase-client/src/main/java/org/apache/hadoop/hbase/client/BufferedMutatorParams.java

```diff
@@ -70,6 +70,15 @@ public int getRpcTimeout() {
     return rpcTimeout;
   }
 
+  public BufferedMutatorParams operationTimeout(final int operationTimeout) {
+    this.operationTimeout = operationTimeout;
+    return this;
+  }
+
+  /**
+   * @deprecated Use {@link #operationTimeout()}
```

Comment:
Please mention the version where we deprecate this method and the version we plan to remove this method. Ithink it should be 2.3.0 and 4.0.0?

Reply:
Looking at the other depreciated methods in `BufferedMutatorParams`, it looks like the version we deprecate this should be `3.0.0`.